docs/contributing.md

```diff
@@ -5,5 +5,31 @@ We welcome all bug reports and pull requests.
 Issues: [github.com/dustinfarris/ember-django-adapter/issues][1]
 
 
+## Local Development
+
+To run ember-django-adapter locally, you will have to use `npm link`.
+
+```
+git clone git@github.com:dustinfarris/ember-django-adapter
+cd ember-django-adapter
+npm i && bower i
+npm link
+```
+
+Then in a test ember-cli project, you will have to install a couple bower dependencies separately.
+These are not actually required in the ember-cli project, but errors will be thrown if they are not
```

Comment:
Suggestion: "not actually required in the **test** ember-cli project"

I had to re-read that sentence twice I think the suggestion is clearer. But it's just a suggestion. I'm good with it either.